students/AndrewMiotke/session05/mailroom.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3
+import os
+import sys
+import time
+import shutil
+from textwrap import dedent # I took this from the solutions code because I thought it was great
+
+# The names come from dogs that are in my office
+donors_list = [("Rufio", [897, 200, 200]),
```

Comment:
By now, you should try to convert this into a dict. It will make it much easier to go through and find if a user exists instead of iterating through the list every time you want to do something.